packages/dart/npt_flutter/lib/features/onboarding/widgets/onboarding_button.dart

```diff
@@ -64,7 +67,7 @@ class _OnboardingButtonState extends State<OnboardingButton> {
       // hours for this tiny UX fix
 
       // TODO: fix localizations
-      await AtOnboardingLocalizations.load(const Locale("en"));
+      await AtOnboardingLocalizations.load(LanguageUtil.getLanguageFromLocale(Locale(Platform.localeName)).locale);
```

Comment:
Onboarding flutter only supports english && french, can you confirm behaviour when our app's default locale is not english?

Reply:
If the device language is not one from the languages npt_flutter supports it will default to English. So it wouldn't even return French because we don't have localization for French. When we update at_onboarding_flutter to support the languages npt_flutter supports, we wouldn't have to change our app code.

Reply:
Okay, thanks for confirming